README.md

```diff
@@ -103,55 +100,39 @@ The DAG file has to be copied to `dags_folder` directory within all the nodes. I
 In order to enable authentication for the Airflow Web UI check the "Enable Airflow Authentication" option. You can create Airflow users using one of two options below.
 
 ### Creating Airflow Users using UI:
-1. Navigate to Airflow CSD. In the configurations page, enter the Airflow Username, Airflow Email, Airflow Password you want to create.
-2. Deploy the client configurations to create the Airflow user.
+One way to add Airflow users to the database is using the `airflow-mkuser` script.  Users can be added as follows:
+
+1. Navigate to Airflow WebUI.
+2. In the Admin dropdown choose Users.
+3. Choose Create and enter the username, email, and password you want to create.
 
 Note: Although the last created user shows up in the Airflow configurations, you can still use the previously created users.
 
-### Using mkuser.sh
-Another way to add Airflow users is using the `mkuser.sh` script.  Users can be added as follows:
-1. Navigate to the current working directory of the CSD under `/var/run/cloudera-scm-agent/process`
-2. Export PYTHONPATH and AIRFLOW_HOME environment variables. By default these are:
+### Using airflow-mkuser
+Another way to add Airflow users to the database is using the `airflow-mkuser` script.  Users can be added as follows:
 
-   PYTHONPATH:
-   ```bash
-   export PYTHONPATH=/opt/cloudera/parcels/AIRFLOW/usr/lib/python2.7/site-packages:$PYTHONPATH
-   ```
-   Airflow Home:
-   ```bash
-   export AIRFLOW_HOME=/var/lib/airflow
-   ```
-3. Within the scripts directory, you can find the `mkuser.py` file. Execute `mkuser.py` to add a user to Airflow:
-   ```bash
-   /opt/cloudera/parcels/AIRFLOW/bin/python2.7 mkuser.py <Username> <UserEmail> <Password>
-   ```
-   For example, this can be like
-   ```bash
-   /opt/cloudera/parcels/AIRFLOW/usr/bin/python2.7 mkuser.py airflowUser airflow@email.com airflowUserPassword
-   ```
+```bash
+airflow-mkuser <username> <email> <password>
+```
+For example, this can be like:
+```bash
+airflow-mkuser admin admin@localdomain password123
+```
 
 ## Building the CSD
 ```bash
 git clone https://github.com/teamclairvoyant/apache-airflow-cloudera-csd
 cd apache-airflow-cloudera-csd
-mvn clean package
-```
-or
-```bash
-java -jar target/validator.jar -s src/descriptor/service.sdl
-jar -cvf AIRFLOW-1.0.0.jar -C src/ .
+make dist
```

Comment:
Add step to update the version and version-parcel file as needed

Reply:
Good catch.

Reply:
Fixed in 5dc81f05720fa813ee61887962049ee707c28caa